src/third-party-tags/inizio.ts

```diff
@@ -0,0 +1,34 @@
+import { GetThirdPartyTag } from '../types';
+
+const onLoad = () => {
+	const handleQuerySurveyDone = (
+		surveyAvailable: boolean,
+		survey: { measurementId: string },
+	) => {
+		if (surveyAvailable) {
+			if (window && window.googletag) {
+				window.googletag.cmd.push(() => {
+					window.googletag.pubads().setTargeting('inizio', 't');
+				});
+			}
+			console.log(`surveyAvailable: ${survey.measurementId}`);
+		}
+	};
+	// eslint-disable-next-line no-underscore-dangle
+	window._brandmetrics = window._brandmetrics || [];
+	// eslint-disable-next-line no-underscore-dangle
```

Comment:
wonder if we should just disable this rule for this codebase? feels like it's going to crop up a lot and it's not really catching anything useful?

Reply:
Defining the type seems to have fixed it anyway. Removed in ed7e573b6b8605ca0743eeaae6492d51f081b5ec